src/wallet/rpcwallet.cpp

```diff
@@ -2823,8 +2827,17 @@ UniValue bumpfee(const JSONRPCRequest& request)
         // new fee rate must be at least old rate + minimum incremental relay rate
         if (nNewFeeRate.GetFeePerK() < nOldFeeRate.GetFeePerK() + ::incrementalRelayFee.GetFeePerK()) {
             nNewFeeRate = CFeeRate(nOldFeeRate.GetFeePerK() + ::incrementalRelayFee.GetFeePerK());
+            if (specifiedConfirmTarget || payTxFee.GetFeePerK() == 0) {
+                // In the default case where the user is depending on the wallet
+                // to smartly pick the fee, the wallet should have its own
```

Comment:
Maybe s/should have its own minimum incremental fee/uses a conservative WALLET_INCREMENTAL_RELAY_FEE value/. Saying "should" could suggest that the wallet doesn't already do this.